public/doc/swagger-2-v0.0.1.yaml

```diff
@@ -14,7 +14,7 @@ securityDefinitions:
 security:
 - UserSecurity: []
 paths:
-  "/container_groups":
+  "/v0.0/container_groups":
```

Comment:
This is  not right...this should be part of the basePath.

Reply:
Reverted